q2_demux/_type.py

```diff
@@ -8,4 +8,9 @@
 
 from qiime2.plugin import SemanticType
 
+# TODO: migrate these to q2-types someday
 RawSequences = SemanticType('RawSequences')
+
+EMPSequences = SemanticType('EMPSequences')
```

Comment:
I think we should call this `EMPSingleEndSequences`. I think this will make more sense now that I'm looking at it.